docs/plugins.md

```diff
@@ -0,0 +1,40 @@
+# Plugins
+
+Since Go1.8, IronFunctions also supports plugins. You can extend it with binary
+plugins that builds on internal bindings covered in [extending.md](operating/extending.md).
+
+In order to use a plugin, make sure the `.so` file is compiled and placed
+somewhere accessible by IronFunctions daemon. When starting it, you will need to
+set up an environment variable stating where the plugin are:
+
+```sh
+PLUGINS="commonlog.so" ./functions
+```
+
+Note that you can activate more than one plugin at a time using comma:
+
+```sh
+PLUGINS="commonlog.so,requestdump.so" ./functions
```

Comment:
Is this the best solution to load plugins?
Couldn't we just add an env var pointing to a directory location from where all (valid) plugins would be loaded?
What do you think?